lua/nvim-tree/explorer/reload.lua

```diff
@@ -62,7 +62,10 @@ function M.reload(node, status)
             table.insert(node.nodes, link)
           end
         end
+
       end
+      local n = nodes_by_path[abs]
+      n.executable = builders.is_executable(abs, n.extension)
```

Comment:
check if n exists first please to make sure that this won't crash.

Reply:
I did. But I removed it before I created PR because I thought line 49 already covered the case.
What do you think?

Reply:
indeed but it still might not go into the conditionals, so make the check again :)

Reply:
Ok, done